core/src/verified_vote_packets.rs

```diff
@@ -245,6 +245,23 @@ impl VerifiedVotePackets {
                             }
                         }
                         _ => {
+                            if let Some(FullTowerVote(gossip_vote)) =
```

Comment:
do we need a similar check to `if slot > latest_gossip_slot {` like from above?

Reply:
good catch, otherwise an old incremental vote resubmitted on feature activation boundary could end up wiping your saved votes.

Reply:
also if `is_full_tower_vote_enabled` is enabled, should we just ignore these incremental votes, or are we assuming any downgrade will require pushing code to revert the replay stage changes anyways?

Reply:
we can gate downgrade on having this flag enabled, and then replace it with the new flag if we need to push downgrade code. so that we don't forget.